sentry/project_filter.go

```diff
@@ -74,11 +74,15 @@ type BrowserExtensionParams struct {
 }
 
 // UpdateBrowserExtensions updates configuration for browser extension filter
-func (s *ProjectFilterService) UpdateBrowserExtensions(organizationSlug string, projectSlug string, active bool) (*http.Response, error) {
-	apiError := new(APIError)
+func (s *ProjectFilterService) UpdateBrowserExtensions(ctx context.Context, organizationSlug string, projectSlug string, active bool) (*Response, error) {
+	url := fmt.Sprintf("0/projects/%v/%v/filters/browser-extensions/", organizationSlug, projectSlug)
 	params := BrowserExtensionParams{active}
-	resp, err := s.sling.New().Put("projects/"+organizationSlug+"/"+projectSlug+"/filters/browser-extensions/").BodyJSON(params).Receive(nil, apiError)
-	return resp, relevantError(err, *apiError)
+	req, err := s.client.NewRequest("PUT", url, params)
```

Comment:
The same thing here: `http.MethodPut`.

Reply:
Let's nudge the author into better practices :P https://github.com/Canva/go-sentry/pull/3/commits/eb3a516a4eb089d8e993bdd6f81a937b21db7df3